integration_tests/auth/conftest.py

```diff
@@ -236,11 +241,33 @@ def admin_token(
         yield token
 
 
+@pytest.fixture
+def member_token(
+    get_new_user: _GetNewUser,
+    member_email: str,
+    member_password: str,
+    log_in: _LogIn,
+) -> Iterator[_Token]:
+    member = get_new_user(UserRoleInput.MEMBER)
+    with log_in(member.profile.password, email=member.profile.email) as (token, _):
+        yield token
```

Comment:
it's already logged in
```suggestion
    yield get_new_user(UserRoleInput.MEMBER).token
```